keylime/dsse/dsse.py

```diff
@@ -8,26 +8,29 @@
 import base64
 import binascii
 import json
+from abc import abstractmethod
 
 # Protocol requires Python 3.8+.
 from typing import Iterable, List, Optional, Tuple
 
 
 class Signer:
-    def sign(self, message: bytes) -> bytes:  # pylint: disable=unused-argument,no-self-use,useless-option-value
+    @abstractmethod
+    def sign(self, message: bytes) -> bytes:  # xpylint: disable=unused-argument,no-self-use,useless-option-value
```

Comment:
this xpylint is just done for disablement and from what it looked like the whole thing pylint comment can be removed -- the other ones below as well